### PR TITLE
Add an internal onboarding guideline

### PR DESCRIPTION
**Pull request by BrianBohe**
How to get privileges to run /azp commands on your team's repos.
How to join Microsoft and .Net foundation organizations.

### CODE REVIEW

**Comment by BrianBohe** [comment]
@jaredpar

**Comment by jkotas** [comment]
https://github.com/dotnet/core/blob/master/Documentation/microsoft-team.md is a very similar onboarding doc. Should we just link to the shared one?

File: docs/workflow/intern-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<h3>Join Microsoft / .Net Foundation organizations</h3>
+
+1. Link Microsoft account with the Github account and join Microsoft GitHub organization:
+    + Go to https://opensourcehub.microsoft.com/link/ and https://repos.opensource.microsoft.com/ to link.
+    + Go to https://repos.opensource.microsoft.com/Microsoft to join Microsoft org.
+    + Go to  https://repos.opensource.microsoft.com/DotNet to join DotNet org.
+    + Join this group as well https://repos.opensource.microsoft.com/dotnet/teams/mlnet if you want to do PR via codeflow.
```
Comment (jkotas) [inline docs/workflow/intern-onboarding.md]: Nit: What does ML.NET has to do with codeflow?

Comment (BrianBohe) [reply docs/workflow/intern-onboarding.md]: I merged the peaces of guidelines I got by interacting with other members for any new contributors (it took me a week to get /azp command privileges). Giving your previous comment, I think this file should only link that document and tells one to ask for the /azp commands privileges.

**Comment by danmosemsft** [comment]
There was a similar PR not long ago and the feedback was that we don't generally include docs that are only  relevant to Microsoft onboarding and not necessarily specific to this repo either. Feels like they should go on a one note or SharePoint or something else internal.

**Comment by BrianBohe** [comment]
> There was a similar PR not long ago and the feedback was that we don't generally include docs that are only relevant to Microsoft onboarding and not necessarily specific to this repo either. Feels like they should go on a one note or SharePoint or something else internal.

I agree with you and we actually have one.

**Pull request closed without merging**